plugins/gatsby-plugin-elasticlunr-search-fiori/gatsby-node.js

```diff
@@ -0,0 +1,194 @@
+"use strict"
+const webVersion = require('.../src/pages/versions/web-version.json')
+const iosVersion = require('.../src/pages/versions/ios-version.json')
+const androidVersion = require('.../src/pages/versions/android-version.json')
+const cuxVersion = require('.../src/pages/versions/android-version.json')
+
+
+function _objectSpread(target) {
+    for (var i = 1; i < arguments.length; i++) {
+        var source = arguments[i] != null ? arguments[i] : {}
```

Comment:
maybe es6 in this block of code const/let/arrows etc. ?

Reply:
done